src/easysocial/search.php

```diff
@@ -201,24 +204,24 @@ public function getFriendList($log_user, $search, $limitstart, $limit)
 	 *
 	 * @since 1.0
 	 */
-	public function getEventList($log_user, $search, $limitstart, $limit)
+	public function getEventList($logUser, $search, $limitstart, $limit)
 	{
 		$mapp = new EasySocialApiMappingHelper;
 		$result = new stdClass;
 
 		$db = JFactory::getDbo();
-		$query1 = $db->getQuery(true);
-		$query1->select($db->quoteName(array('cl.id')));
-		$query1->from($db->quoteName('#__social_clusters', 'cl'));
+		$query = $db->getQuery(true);
```

Comment:
@Pratikshakhandagale Do we need separate API for search? You can perform this action in the models itself.

cc
@coolbung

Reply:
@thite-amol , 
Do we need separate API for search? 
- No, we don't need to separate api for search because it is not much the complex code. let me know if there is any other reason to separate api. 

You can perform this action in the models itself.
- Yes, we can achieve this by using the ES method, I have added this task in PMS, I will do this.